client/clienterror/clienterror.go

```diff
@@ -131,6 +131,11 @@ func New(err error) *APIError {
 			ae.ErrorDetails += "Details: " + modifyClusterFailedErr.Payload.Message
 		}
 
+		if modifyClusterFailedErr.Payload.Code == "INVALID_INPUT" {
```

Comment:
This problem was noted here: https://github.com/giantswarm/giantswarm/pull/2206#issuecomment-457584528

Output now looks like this:
```
 gsctl scale cluster 5nkxf --workers-max 4
The cluster size is currently pinned to 3.
Do you want to change the limits to be min=3, max=4? [y/n]: y
Invalid parameters
The cluster could not be updated. workers-min and workers-max must be equal without autoscaling.
```